python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py

```diff
@@ -355,6 +355,13 @@
                     }
                 }
             }
+            partitionDefinition {
+                name
+                dimensionTypes {
+                    dynamicPartitionsDefinitionName
```

Comment:
so always use `dynamicPartitionsDefinitionName` if available otherwise `name` ?

Reply:
`dynamicPartitionsDefinitionName` should only be used in the add partitions mutation, `name` should be used everywhere where we display the dimension names.

Since it's possible that somebody could do something like this:
```
MultiPartitionsDefinition({"a": DynamicPartitionsDefinition("foo"), "b": DynamicPartitionsDefinition("foo")})
```